erpnext/accounts/doctype/payment_reconciliation/payment_reconciliation.py

```diff
@@ -166,6 +179,40 @@ def reconcile(self, args):
 		msgprint(_("Successfully Reconciled"))
 		self.get_unreconciled_entries()
 
+	def reconcile_invoices(self,args):
+		#reconciles entries in invoices table
+
+		for e in self.get('invoices'):
+			e.payment_jv_type = None
+			if e.payment_jv_number and " | " in e.payment_jv_number:
+				 e.payment_jv_type, e.payment_jv_number = e.payment_jv_number.split(" | ")
+
```

Comment:
Shouldn't `self.get_invoice_entries()` be called from here as well? Currently `self.add_invoice_entries` is called only when `order_by` field is selected